app/test_main.py

```diff
@@ -1,3 +1,34 @@
 from app.main import get_human_age
+import pytest
 
-# write your code here
+
+@pytest.mark.parametrize("cat_age, dog_age, output", [
+    (0, 0, [0, 0]),
+    (14, 14, [0, 0]),
+    (15, 15, [1, 1]),
+    (27, 27, [2, 2]),
+    (28, 28, [3, 2]),
+    (28, 20, [3, 1]),
```

Comment:
Add test cases to test scenarios when values for `cat_age` and `dog_age` differ.